INSTALL_NOTES.md

```diff
@@ -411,4 +414,5 @@ You can check you're limits for FD via either `ulimit -n` or, in some case `ulim
 Next...
 ```
 protontricks 359320 dotnet40 win7
```

Comment:
Requires ProtonTricks installed. I'll make an update to include the installation of ProtonTricks as well... My bad.